### PR TITLE
New root features

### PR DESCRIPTION
**Pull request by jtchk**
`getSolutions()` will be a new python function which will return a python list of the solutions generated by a Block. 
The inputs are the name of the block, as specified in the Lua file, in the form of a python string and the python list of `Particle`'s of final state particles. The evaluation of the weight is necessary s the MET will be computed beforehand. 

Several changes are incorporated in order to make MoMEMta consistent with the new versions of ROOT (>=6.20.00).

### CODE REVIEW

File: core/src/python/bindings.cc [hunk 1]
```diff
@@ -100,6 +101,48 @@ bp::list MoMEMta_computeWeights(MoMEMta& m, bp::list particles) {
     return MoMEMta_computeWeights_MET(m, particles, bp::list());
 }
 
+bp::list MoMEMta_getSolutions(MoMEMta& m, const std::string& blockName, bp::list particles_) {
+
+    /* Extract the missing transverse Energy */
+    bp::list met_;
```
Comment (swertz) [inline core/src/python/bindings.cc:107]: I don't understand how the MET is handled.

For the regular particles, you retrieve them from the `particles_` python list. But the `met_` list is just created here, while I think it should also be passed as an argument to the function?

File: core/src/python/bindings.cc [hunk 2]
```diff
@@ -100,6 +101,48 @@ bp::list MoMEMta_computeWeights(MoMEMta& m, bp::list particles) {
...
+    auto weights = m.computeWeights(particles, met);
+
+    /* Define the address of the solutions of the required block */
+    InputTag solInputTag {blockName, "solutions"};
+
+    /* Retrieve solutions out of the memory pool */
+    auto sol = m.getPool().get<SolutionCollection>(solInputTag);
+
+    /* Create the converter to create a Boost Python 4-momentum out of a LorentzVector */
+    /*LorentzVector_to_python four_momentum_converter;*/
```
Comment (swertz) [inline core/src/python/bindings.cc:129]: Can you remove this if it isn't used?

**Review by swertz — CHANGES_REQUESTED** [review]
Thanks!

Could you just please add a meaningful description for your pull request?

Can you remove the empty `build.log` file from the PR?

Can you also bump the minimally supported ROOT version to 6.20.00 at these two places:
- https://github.com/MoMEMta/MoMEMta/blob/master/CMakeLists.txt#L86
- https://github.com/MoMEMta/MoMEMta/blob/master/README.md

File: core/src/python/bindings.cc (hunk 2 above)
Comment (jtchk) [reply core/src/python/bindings.cc:129]: Lines 128-129 have been deleted from the `bindings.cc`file.

**Comment by swertz** [comment]
Thanks again!

**Review by swertz — APPROVED** [review]
LGTM

**Pull request merged**